README.md

```diff
@@ -44,4 +44,4 @@ We welcome any contributions. See the [CONTRIBUTING](https://github.com/textureg
 
 ## License
 
-The Texture project was created by Pinterest as a continuation, under a different name and license, of the AsyncDisplayKit codebase originally developed by Facebook.  AsyncDisplayKit was originally released by Facebook under a BSD license and additional patent grant.  Those BSD and patent licenses govern use of code in Texture contributed prior to 4/13/2017 (the original AsyncDisplayKit code), and copies of the licenses are included in the root directory of this source tree for reference.  All code contributed to Texture after 4/13/2017 is released by Pinterest under an Apache 2.0 license.
+The Texture project is available for free use, as described by the Apache 2.0 license. Texture is based on code from the AsyncDisplayKit project originally developed by Facebook. Although AsyncDisplayKit was originally released by Facebook under a BSD license and additional "PATENTS v2" grant, the entire Texture codebase is now under a unified Apache 2 license with approval from Facebook to remove the PATENTS v2 file (including for the AsyncDisplayKit fork from 4/13/17).
```

Comment:
Not sure we need the second sentence any longer. I think you can remove all references to the old license.